crates/executor/state_handler/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "ixc_state_handler"
+version = "0.1.0"
+edition = "2021"
+repository.workspace = true
+license.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+ixc_message_api = { path = "../../module_system/message_api" }
+allocator-api2 = "0.2.18"
```

Comment:
we should probably use workspace dependencies to track these versions going forward